docs/example_pipelines/iris/iris-pipeline.yaml

```diff
@@ -0,0 +1,223 @@
+
+          '
+        - "\nimport kfp\nfrom kfp import dsl\nfrom kfp.dsl import *\nfrom typing import\
+          \ *\n\ndef create_dataset(iris_dataset: Output[Dataset]):\n    import pandas\
+          \ as pd\n\n    csv_url = 'https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data'\n\
+          \    col_names = [\n        'Sepal_Length', 'Sepal_Width', 'Petal_Length',\
+          \ 'Petal_Width', 'Labels'\n    ]\n    df = pd.read_csv(csv_url, names=col_names)\n\
+          \n    with open(iris_dataset.path, 'w') as f:\n        df.to_csv(f)\n\n"
+        image: quay.io/opendatahub/ds-pipelines-iris-base:v1.0
+    exec-normalize-dataset:
+      container:
+        args:
+        - --executor_input
+        - '{{$}}'
+        - --function_to_execute
+        - normalize_dataset
+        command:
+        - sh
+        - -c
+        - "\nif ! [ -x \"$(command -v pip)\" ]; then\n    python3 -m ensurepip ||\
+          \ python3 -m ensurepip --user || apt-get install python3-pip\nfi\n\nPIP_DISABLE_PIP_VERSION_CHECK=1\
+          \ python3 -m pip install --quiet --no-warn-script-location 'kfp==2.6.0'\
+          \ '--no-deps' 'typing-extensions>=3.7.4,<5; python_version<\"3.9\"' && \"\
+          $0\" \"$@\"\n"
+        - sh
+        - -ec
+        - 'program_path=$(mktemp -d)
+
+
+          printf "%s" "$0" > "$program_path/ephemeral_component.py"
+
+          _KFP_RUNTIME=true python3 -m kfp.dsl.executor_main                         --component_module_path                         "$program_path/ephemeral_component.py"                         "$@"
+
+          '
+        - "\nimport kfp\nfrom kfp import dsl\nfrom kfp.dsl import *\nfrom typing import\
+          \ *\n\ndef normalize_dataset(\n    input_iris_dataset: Input[Dataset],\n\
+          \    normalized_iris_dataset: Output[Dataset],\n    standard_scaler: bool,\n\
+          \    min_max_scaler: bool,\n):\n    if standard_scaler is min_max_scaler:\n\
+          \        raise ValueError(\n            'Exactly one of standard_scaler\
+          \ or min_max_scaler must be True.')\n\n    import pandas as pd\n    from\
+          \ sklearn.preprocessing import MinMaxScaler\n    from sklearn.preprocessing\
+          \ import StandardScaler\n\n    with open(input_iris_dataset.path) as f:\n\
+          \        df = pd.read_csv(f)\n    labels = df.pop('Labels')\n\n    if standard_scaler:\n\
+          \        scaler = StandardScaler()\n    if min_max_scaler:\n        scaler\
+          \ = MinMaxScaler()\n\n    df = pd.DataFrame(scaler.fit_transform(df))\n\
+          \    df['Labels'] = labels\n    normalized_iris_dataset.metadata['state']\
+          \ = \"Normalized\"\n    with open(normalized_iris_dataset.path, 'w') as\
+          \ f:\n        df.to_csv(f)\n\n"
+        image: quay.io/opendatahub/ds-pipelines-iris-base:v1.0
```

Comment:
Testing this pipeline, I see an issue with accessing this image. Is it private?
 
![Screenshot from 2024-02-12 18-13-45](https://github.com/opendatahub-io/data-science-pipelines-operator/assets/38726729/1f41bccd-8168-4534-961b-517949c04504)

Reply:
This image isn't built yet. @HumairAK could you please copy over your image and build this one? Thanks!